definitions/procedures/qpid_queues_clear.rb

```diff
@@ -0,0 +1,17 @@
+class Procedures::QpidQueuesClear < ForemanMaintain::Procedure
+  metadata do
+    for_feature :qpid_queues
+    description 'clear qpid queues'
+  end
+
+  def run
+    with_spinner('clear qpid queues') do |spinner|
+      feature(:katello_service).make_stop(spinner, '--exclude qpidd')
```

Comment:
We should remember the original state of the service, and start only if it was running before at the end. What about having something like this:

```ruby
def make_stop(spinner, args) do
   originally_runner_services = running_services
   execute!('katello-service stop #{args}')
   yield
 ensure
   execute!('katello-service start --only #{originally_running_services}')
end
```